rpcclient/examples/customcommand/README.md

```diff
@@ -0,0 +1,32 @@
+Namecoin Core Custom Command Example
```

Comment:
I'm fine with leaving the other parts of the readme as is and then update with a future PR, but I think we should fix the name at least so it isn't `Namecoin`

Reply:
Let's do this in a separate PR. I think just using "Custom Command Example" is appropriate.

Reply:
Fixed.